app/views/finders/show.html.erb

```diff
@@ -7,6 +7,9 @@
   <%= auto_discovery_link_tag(:atom, signup_links[:feed_link]) %>
   <%= render 'finder_meta', content_item: content_item %>
 <% end %>
+<% content_for :meta_tags do %>
```

Comment:
Wrong view file, `<%= brexit_question_variant.analytics_meta_tag.html_safe %>` would go in [`brexit_checker/show.html.erb#L5`](https://github.com/alphagov/finder-frontend/blob/master/app/views/brexit_checker/show.html.erb#L5) within the existing `:head` section. Otherwise the change looks great and works as expected.

Reply:
Thanks Alex!